### ISSUE TITLE
Several

### ISSUE THREAD

**Issue by HAXEK** [florisboard/florisboard#1175]
✅Suggestions:

 ➡️Enable some option to keep the suggested gestures, plus the smart bar at the same time. 

➡️Is it possible to enable any option to use only suggestion from the internal user dictionary? 

➡️Add Spanish please.

➡️Add emoji finder. 

Cheer up and greetings from Venezuela! 🇻🇪

**Comment by Glitchy-Tozier** [comment]
Hi and thank you for taking the time to write this issue.
- could you elaborate on number 1?
- 3&4 already have been suggested (3->many many times, 4->#45), so I suggest removing them/crossing them out to keep redundancy low.

**Comment by HAXEK** [comment]
➡️Enable some option to keep the suggested gestures, plus the smart bar at the same time.
![1629631715223](https://user-images.githubusercontent.com/89304575/130353468-ed061039-a402-4125-a23b-8647eea9306e.jpg)

**Comment by hbarsaiyan** [comment]
> ➡️Enable some option to keep the suggested gestures, plus the smart bar at the same time.
> ![1629631715223](https://user-images.githubusercontent.com/89304575/130353468-ed061039-a402-4125-a23b-8647eea9306e.jpg)

![Screenshot_20210823-000612_Bromite](https://user-images.githubusercontent.com/37732050/130366885-72938a4b-49b6-48bd-8cb8-f589f1a23abe.png)
Swiftkey has a switch on the left as you can see for showing the smartbar on top of suggestions. Probably something like that could be done in florisboard.

**Comment by HAXEK** [comment]
thanks it would be great.😃👍

**Comment by patrickgold** [comment]
> Enable some option to keep the suggested gestures, plus the smart bar at the same time.

Multiple smartbars/stacked smartbars are already planned and have been suggested but I cannot find the issue for it atm.

> Is it possible to enable any option to use only suggestion from the internal user dictionary?

Currently not, no. This is a unique idea (haven't seen a suggestion for it yet). Can definitely consider adding such feature sometime in the future, maybe 0.3.15 with the suggestions rework.

I will rename this issue's title so the internal dictionary suggestion is the primary topic and will assign it to the correct area,

**Comment by Joshix-1** [comment]
Only using the internal user dictionary would be great. 

It would be also great if you could fill it while typing. So that every word you type gets added to a dictionary that fills over time. With that it would be easier to fill your own dictionary.

**Comment by Gitoffthelawn** [comment]
> Only using the internal user dictionary would be great.
> 
> It would be also great if you could fill it while typing. So that every word you type gets added to a dictionary that fills over time. With that it would be easier to fill your own dictionary.

Yes!  Most people use a tiny subset of words of the language in which they are writing.  I've read review after 1 star review of Google and Microsoft keyboard apps that recommend words that the writer would never choose to use.

I would even go so far as to suggest different word dictionaries for different apps (for example, when SMS texting, you likely use different words than when writing a business letter).  I realize that might be a long way off, but it will likely benefit many.
